swift/WorkflowUI/Sources/Container/UIKitRenderableViewController.swift

```diff
@@ -0,0 +1,81 @@
+import UIKit
+import ReactiveSwift
+
+
+/// Internal helper to display the current screen within a ContainerViewController
+internal final class UIKitRenderableViewController: UIViewController {
+
+    private(set) internal var currentScreenViewController: AnyScreenViewController? = nil
+
+    private let viewRegistry: ViewRegistry
+
+    init(viewRegistry: ViewRegistry) {
+        self.viewRegistry = viewRegistry
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
```

Comment:
This `viewDidLoad` doesn't need to be over-ridden (since it's a no-op)

Reply:
Done